torch/onnx/utils.py

```diff
@@ -506,7 +509,15 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
                                   .format(op_name, op_name))
                     return None
                 fn = getattr(torch.onnx.symbolic, op_name)
-                return fn(g, *inputs, **attrs)
+                try:
+                    return fn(g, *inputs, **attrs)
+                except AssertionError:
+                    if is_aten_fallback_export:
+                        # Falling back to ATen
+                        warnings.warn("An assertion error occured when trying to export {}. Falling back to ATen"
+                                      .format(op_name))
```

Comment:
Makes sense to add error's text as well.

Reply:
You're right. The text has been added.